How to distribute an extension in the Chrome Web Store (#1854)
GoogleChrome/developer.chrome.com: site/en/docs/extensions/mv3/getstarted/index.md

}
```

### Load unpacked extension {: #unpacked }

The directory holding the manifest file can be added as an extension in developer mode in its

> current state. To load an unpacked extension in developer mode, follow the next steps:

Reviewer: ```suggestion
current state. To load an unpacked extension in developer mode, follow these steps:
```
